File: atcoder/2020/ABC/0216_abc155/C.py
n = int(input())
st = {}

maxi = 0

for _ in range(n):
    s = input()

    if s in st:
        st[s] += 1
    else:
        st[s] = 1

    if st[s] > maxi:
        maxi = st[s]

ans = []
for k, v in st.items():
    if v==maxi:
        ans.append(k)

ans = sorted(ans)
for a in ans:
    print(a)

File: atcoder/2020/ABC/0216_abc155/C.js
function Main(input) {
  inp = input.split("\n")
  let N = parseInt(inp[0]);
  let ans = {};
  let maximum = 0;
  for (let i = 0; i < N; i++) {
    S = inp[i+1];
    if (ans[S]) {
      ans[S] += 1;
      maximum = Math.max(maximum, ans[S]);
    } else {
      ans[S] = 1;
      maximum = Math.max(maximum, ans[S]);
    }
  }

  let ans2 = [];
  for (let key in ans) {
    let value = ans[key];
    if (value === maximum) {
      ans2.push(key);
    }
  }

  ans2.sort();
  for (let i = 0; i < ans2.length; i++) {
    console.log(ans2[i]);
  }

};

Main(require("fs").readFileSync("/dev/stdin", "utf8"));